### PR TITLE
[RW-5778][risk=no]Anchor graph showing version dates under each survey question

### PR DESCRIPTION
**Pull request by mhigbyflowers**
image

### CODE REVIEW

File: public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      followPointer: true,
+      headerFormat: '<b>{point.x}</b><br/>',
+      pointFormat: '{series.name}: {point.y}<br/>Total: {point.stackTotal}'
+    };
+    this.chartOptions.colors = this.colors;
+  }
+
+  public sortAnswers(answers: any[]) {
+    const result = answers.reduce((r, a) => {
+      r[a.stratum7] = [...r[a.stratum7] || [], a];
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.ts:92]: 🤷‍♂️

File: public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    justify-content: center;
+    align-items: flex-end;
+}
+.legend {
+    padding:1rem;
+    padding-bottom: 2rem;
+}
+
+.legend-inner {
+    overflow-y: scroll;
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.css:13]: image
Can this be set to auto to not have the scroll bar when not needed ?

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good to me. Thanks.

File: public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.css (hunk 2 above)
Comment (mhigbyflowers) [reply public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.css:13]: thanks. I will fix that in the tool tip PR

**Pull request merged**